.github/workflows/ci.yml

```diff
@@ -512,7 +512,7 @@ jobs:
         include:
           - test_name: demo-indirect-invocation
           - test_name: demo-direct-call
-          - test_name: demo-sidechain
+          # - test_name: demo-sidechain
```

Comment:
:nail_care: let's not leave commented-out ghosts please ;) If we intend to revive this at some point then let's document that in a followup issue.